src/scss/_base.scss

```diff
@@ -1,20 +1,37 @@
 * {
+  line-height: calc(10px + 2ex);
+}
+
+*,
+::before,
+::after {
   box-sizing: border-box;
 }
 
+html,
 body {
-  background-color: $white;
-  min-width: 320px;
-  position: relative;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
+  height: 100%;
+}
+
+body {
+  color: var(--color-text);
+  font-size: var(--font-size-lg);
+  overflow-x: hidden;
 }
 
-img {
+img:not(.dark-bg *, .footer *, .blog-posts *),
```

Comment:
Be aware of the compatibility of selector lists within `:not()` https://caniuse.com/css-not-sel-list

It's a newer thing than just a single selector.